docs/getting-started/install-endpoint.asciidoc

```diff
@@ -65,17 +65,23 @@ If you have upgraded to an {stack} version that includes {fleet-server} 7.13.0 o
 [role="screenshot"]
 image::images/install-endpoint/add-agent.png[Add agent flyout on the Fleet page.]
 
-. Select an agent policy for the {agent}. You can select an existing custom policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+. Select an agent policy for the {agent}. You can select an existing policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
 +
-The selected policy should include {endpoint-sec}.
+The selected agent policy should include {endpoint-sec}.
 +
 [role="screenshot"]
 image::images/install-endpoint/endpoint-configuration.png[Add agent flyout with Endpoint Security integration highlighted.,575]
 
-. Download and extract the {agent} package on your host's machine.
+. Ensure that the **Enroll in {fleet}** option is selected. {endpoint-sec} cannot be integrated with {agent} in standalone mode.
 
-. Open a command-line interface and navigate to the directory where the {agent} is located. Copy and run the command from {fleet} for your OS to enroll and run the agent.
+. Select the appropriate platform or operating system for the host, then copy the provided commands.
 
-. After you have enrolled the {agent} on your host, select **Close**. The host now appears on the **Endpoints** page in the {security-app}. It may take a minute or two while the {agent} starts up.
+. On the host, open a command-line interface and navigate to the directory where you want to install {agent}. Paste and run the commands from {fleet} to download, extract, enroll, and start {agent}.
+
+. (Optional) Return to the **Add agent** flyout in {fleet}, and observe the **Confirm agent enrollment** and **Confirm incoming data** steps automatically checking the host connection. It may take a few minutes for data to arrive in {es}.
+
+. After you have enrolled the {agent} on your host, you can click **View enrolled agents** to access the list of agents enrolled in {fleet}. Otherwise, select **Close**. 
++
+The host will now appear on the **Endpoints** page in the {security-app}. It may take another minute or two for endpoint data to appear in {elastic-sec}.
```

Comment:
```suggestion
The host will now appear on the **Endpoints** page in the {security-app}. It may take a moment for endpoint data to appear in {elastic-sec}.
```

Reply:
Super tiny nit here and really, both work. I mainly made this suggestion to obscure performance benchmarks since (I assume) it'd vary based on the user's situation.

Reply:
@nastasha-solomon I get your point, but I think it's better to prepare the reader for a longer wait than just a "moment," so they don't get frustrated or think it's not working. The Fleet UI was updated to say "It may take a few minutes for data to arrive in Elasticsearch" and it's a similar delay for the data to then reach the Security app.

Reply:
@joepeeples ah, ok I didn't know the Fleet UI had similar language. Also, I see your point that transparency isn't necessarily "bad" in this case and can actually help our users set expectations when using the Security app. Another benefit of quantifying the wait time is that users will be able to recognize what an abnormally long wait time looks like.